docs/src/developer/reference/config-options.md

```diff
@@ -372,6 +372,48 @@ To get the public key run:
 openssl ec -in private.pem -pubout --out public.pem
 ```
 
+### Channels
+
+Channels are configured per team (as "team feature".) The configuration defines
+if the channels feature is activated and who can create or open channels.
+
+The configuration has these fields:
+
+- `lockStatus`: `locked` or `unlocked`
+- `status`: `enabled` or `disabled`
+- `config.allowed_to_create_channels`: _roles_
+- `config.allowed_to_open_channels`: _roles_
+
+Possible _roles_ are:
+
+| Value          | Who is that?                                    | UX description                    |
+| -------------- | ----------------------------------------------- | --------------------------------- |
+| `everyone`     | Partner (a.k.a. external), Member, Admin, Owner | Everyone in the team              |
+| `team-members` | Member, Admin, Owner                            | Team members (excluding external) |
+| `admins`       | Admin, Owner                                    | Only admins                       |
```

Comment:
wayyy too late in the process, but why didn't we just use the roles we already have here?  they way they are implemented at the moment, they all contain each other, so "partner" would behave identically to "everyone", "member" to "team-members", and "admin" to "admins".  (and we could also set it to "owner" if we really wanted to.)

Reply:
IIRC these values were stated in the RFC. Now, they are described in the Tech Spec: https://wearezeta.atlassian.net/wiki/spaces/ENGINEERIN/pages/1758134359/Channels+feature+flag+and+config

Internally, these `ChannelPermissions` _should_ be mapped to `Roles` before we use them for any logic.

(BTW - This was the advice from @pcapriotti I told you about: Keep the `ToSchema` instances close to the expected JSON format. That's why there's no internal mapping to e.g. `Role`s in the `ToSchema` instance.)